pkg/cmd/stack/deploy.go

```diff
@@ -124,7 +124,7 @@ func (sd *Stack) deployCompose(ctx context.Context, s *model.Stack, options *Dep
 	output := fmt.Sprintf("Deploying compose '%s'...", s.Name)
 	cfg.Data[statusField] = progressingStatus
 	cfg.Data[outputField] = base64.StdEncoding.EncodeToString([]byte(output))
-	if err := configmaps.Deploy(ctx, cfg, s.Namespace, sd.K8sClient); err != nil {
+	if err := configmaps.Deploy(ctx, cfg, options.Namespace, sd.K8sClient); err != nil {
```

Comment:
We need to pass the `options.Namespace` from the deploy file which is not done currently